### ISSUE TITLE
Time Range button: Name is not defined for ‘Next month’ & ‘previous month’ buttons. 

### ISSUE THREAD

**Issue by msftedad** [grafana/grafana#86656]
### What happened?

Screen reader is just narrating as ‘Button’ for ‘Next’ & ‘Previous’ buttons present for the month.  It is just announcing as button.

<img width="960" alt="code snippet-Name is not defined for ‘Next month’   ‘previous month’ buttons" src="https://github.com/grafana/grafana/assets/93735775/6c8bd919-830c-43ae-93d0-bd28f0c360d4">

https://github.com/grafana/grafana/assets/93735775/e47fe75f-b2ba-424e-80e8-71e1cad1048c

### What did you expect to happen?

Respective name should be defined for the ‘Next’ & ‘Previous’ buttons present for the month. 

Ex: Name should be defined as ‘Go to previous month xxxxxx’ for ‘<’ button. 

### Did this work before?

No

### How do we reproduce it?

1. Open the URL [Grafana Play Home - Dashboards - Grafana](https://play.grafana.org/d/000000012/grafana-play-home?orgId=1) in edge Dev browser.
2. ‘Grafana Play Home’ page will open. 
3. Navigate to ‘Time range’ button & activate it. 
4. Navigate to ‘Date pickers’ present for ‘From’ & ‘To’ fields & activate it. 
5. Turn on narrator using ‘Ctrl+Windows+Enter’ keys. 
6. Navigate to ‘Next’ & ‘Previous’ buttons present for the month & observe the screen reader narration. 

### Environment (with versions)?

Grafana: v11.1.0-69372 (51da96d94e)
OS: 24H2 (OS Build 26100.1) 
Browser: Edge version 125.0.2506.0 (Official build) dev (64-bit)

### Grafana platform?

None
